Character.py

```diff
@@ -1,18 +1,54 @@
 import random
+import json
 
 class Character:
     def __init__(self):
-        self.Name = random.choice(['Kevin', 'Alice', 'Dri', 'J'])
+        #self.Name = random.choice(['Kevin', 'Alice', 'Dri', 'J'])
         pass
 
+
+    def isProficient(self, skill):
+        for k in self.Proficiencies:
```

Comment:
Instead of looping through all of these things, you can just use the keyword 'in'

Reply:
ex
`return skill in self.Proficiencies`